docs/source/06_nodes_and_pipelines/03_modular_pipelines.md

```diff
@@ -4,406 +4,267 @@
 
 In many typical Kedro projects, a single (“main”) pipeline increases in complexity as the project evolves. To keep your project fit for purpose, we recommend that you create modular pipelines, which are logically isolated and can be reused. Modular pipelines are easier to develop, test and maintain, and are portable so they can be copied and reused between projects.
 
-## How do I create a modular pipeline?
-
-You can use a [project-specific CLI command](../09_development/03_commands_reference.md#kedro-commands) to create a modular pipeline. The pipeline name must adhere to [generic Python module naming rules](https://realpython.com/python-pep8/#naming-conventions):
-
-* Can only contain alphanumeric characters and underscores (`A-Za-z0-9_`)
-* Must start with a letter or underscore
-* Must be at least 2 characters long
-
+Modular pipelines allow you to instantiate pipelines multiple times, but support overriding inputs/outputs/parameters. They are reusable within the same codebase and shareable across projects via [micro-packaging](04_micro_packaging.md). This is the modern way to use Kedro and will change the way you think about your pipelines.
```

Comment:
"Modular pipelines allow you to instantiate pipelines multiple times, but support overriding inputs/outputs/parameters."

Shouldn't it be `and` instead of `but`?